lib/levels/enemies/enemies-one.js

```diff
@@ -1,10 +1,13 @@
 const SimpleEnemy = require('../../simple-enemy');
-const QuickEnemy = require('../../quick-enemy');
 
-var enemiesOne = [
-  new SimpleEnemy({x: -100, y: 200}),
-  new SimpleEnemy({x: -200, y: 200}),
-  new SimpleEnemy({x: -300, y: 200})
-];
+let enemiesOne = {
+  createEnemies: function() {
+    return [
+      new SimpleEnemy({ x: -100, y: 200 }),
+      new SimpleEnemy({ x: -200, y: 200 }),
+      new SimpleEnemy({ x: -300, y: 200 })
+    ];
+  }
+};
```

Comment:
Alright so my conclusion is this: JS is weird. If I wrap the enemy creation in a function and return the array, it works. So I'm going to update our other enemy files to match this. However, it would still be great to understand WHY this works over the other method.